dev/com.ibm.ws.jpa_22_fat/bnd.bnd

```diff
@@ -16,10 +16,12 @@ javac.source: 1.8
 
 src: \
 	fat/src,\
+	test-applications/cdi/src,\
 	test-applications/jpa21bootstrap/src,\
 	test-applications/jpa22bootstrap/src
 
 test.project: true
+tested.features=cdi-2.0
```

Comment:
tested.features=cdi-2.0 should be tested.features:cdi-2.0

Reply:
Done.